zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/monitoring/TimetableBackupBean.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zookeeper.server.backup.monitoring;
+
+import org.apache.zookeeper.jmx.ZKMBeanInfo;
+
+public class TimetableBackupBean implements ZKMBeanInfo, TimetableBackupMXBean {
+  public static final String TIMETABLE_BACKUP_MBEAN_PREFIX = "Timetable_Backup_server";
```

Comment:
```suggestion
  public static final String TIMETABLE_BACKUP_MBEAN_PREFIX = "TimetableBackup_id”";
```
Can we use the same convention as discussed in ZooSave meeting on 23rd March 2021?

> Register BackupBean as child bean of JMXLocalPeerBean
> ...
> If this change is made, the BackupBean name should be “Backup”; and if it’s not going to be done, the BackupBean name should be “Backup_id{serverId}”.

Reply:
If it is registered as child bean of BackupBean, then we can just make the MBean name "TimetableBackup". BackupBean would already have the id in its name